pyproject.toml

```diff
@@ -53,5 +53,5 @@ ignore = [
     "SIM117", # flake8-simplify - some of merged with statements are not looking great with black, reanble after drop python 3.9
 ]
 
-target-version = "py38"
+target-version = "py312"
```

Comment:
I think this should be the minimum supported version, which right now is py310?

Reply:
Ok, will change. Minimum python version will be python 3.9 after dropping python 3.8